components/org.wso2.carbon.identity.sso.saml/src/main/java/org/wso2/carbon/identity/sso/saml/builders/SingleLogoutMessageBuilder.java

```diff
@@ -109,6 +88,37 @@ public LogoutRequest buildLogoutRequest(String subject, String sessionId, String
         return logoutReq;
     }
 
+    public LogoutRequest buildLogoutRequest(String destination, String tenantDomain, String sessionId, String subject,
```

Comment:
Add comments for the public methods

Reply:
Fixed with https://github.com/wso2-extensions/identity-inbound-auth-saml/pull/208/